src/app/Main/Body/QueryPage/Results/TableResults/QueryPageContent-TableResultsLayout.component.tsx

```diff
@@ -41,7 +41,7 @@ export interface TableResultsProps {
 export function QueryPageContentTableResultsLayoutComponent(
     props: TableResultsProps
 ): JSX.Element {
-    const [isSumRowColState, setIsSumRowCol] = useState<SumOptions>({
+    const [sumOptionsState, setSumOptionsState] = useState<SumOptions>({

```

Comment:
What's your convention for naming states? sumOptions or sumOptionsState?

Reply:
I had to change the name of the State. Otherwise it is complaining in the "handleSumOptionsChange = (sumOptions: SumOptions)" that sumOptions are already declared. That's why I added the extention "State".

Reply:
Ok, that's a questionable linter rule. ``no-shadow``. Normally the function parameters are local to the functions and hide the outer variables on conflict. I personally use in my everyday work the same names for local variables if it makes sense. It is also common to other languages. The linter rule makes only sense because it can prevent bugs. So it is your decision to disable the rule or name local variables differently.

In general it is better to stick to naming conventions an find another name for the conflicting variable. The priority would then to rename the more local variables, e.g. the function parameter could be named ``changedSumOptions`` and the more global variable would keep its name and stick to the naming convention: ``sumOptions = useState``

Reply:
Done.